src/components/common/Extensions/UserHub/partials/BalanceTab/partials/StreamsInfoRow/hooks.ts

```diff
@@ -73,20 +98,56 @@ export const useStreamingPaymentsTotalFunds = () => {
         totalClaimed,
         isAtLeastOnePaymentActive,
         ratePerSecond: ratePerSecondValue,
+        itemsCountedToAverage,
       } = await calculateTotalsFromStreams({
         streamingPayments: items,
         currentTimestamp: Math.floor(blockTime ?? Date.now() / 1000),
         currency,
         colony,
       });
 
+      const averageStreamingPerMonth = calculateAverageStreamingPayment(
+        new Date(blockTime ?? Date.now() / 1000),
```

Comment:
This stood out to me, you are creating a date object with value in seconds which ends up representing a completely wrong date (some time in 1970):

<img width="414" alt="image" src="https://github.com/user-attachments/assets/fac88b94-efb8-4c7a-a2d6-6ebdad5516d7">

Then in the util, you convert the Date to number to multiply it by 1000 to bring it to the correct units.

Did you mean to either:
- Pass the timestamp as a number representing seconds
- Pass the correct Date object and not modify it further inside your util